modules/example/src/smithy4s/example/Letters.scala

```diff
@@ -7,21 +7,21 @@ import smithy4s.ShapeId
 import smithy4s.ShapeTag
 import smithy4s.schema.Schema.enumeration
 
-sealed abstract class Letters(_value: String, _name: String, _intValue: Int) extends Enumeration.Value {
+sealed abstract class Letters(_value: String, _name: String, _intValue: Int, _hints: Hints) extends Enumeration.Value {
   override val value: String = _value
   override val name: String = _name
   override val intValue: Int = _intValue
-  override val hints: Hints = Hints.empty
+  override val hints: Hints = _hints
   @inline final def widen: Letters = this
 }
 object Letters extends Enumeration[Letters] with ShapeTag.Companion[Letters] {
   val id: ShapeId = ShapeId("smithy4s.example", "Letters")
 
   val hints : Hints = Hints.empty
 
-  case object A extends Letters("a", "A", 0)
-  case object B extends Letters("b", "B", 1)
-  case object C extends Letters("c", "C", 2)
+  case object A extends Letters("a", "A", 0, Hints(smithy.api.EnumValue(smithy4s.Document.fromString("a"))))
```

Comment:
I think we could filter out the `smithy.api.EnumValue` trait from hints rendering considering it's captured as static information in the enumeration.

Reply:
yeah, I was wondering if you'd mention that

Reply:
that'd also have to be done for dynamic. I'll get to it later